schemas/oic.types-schema.json

```diff
@@ -32,6 +32,12 @@
           "description": "A string instance is valid against this attribute if it contains an integer in the range [0, (2**64)-1]",
           "type": "string",
           "pattern": "^0|([1-9][0-9]{0,19})$"
+        },
+        "locn-room-list": {
```

Comment:
Where are these room list locations used?  I don't see it called out in the CR but I think that it used to be there.